### DISCUSSION TITLE
how to use other server, not nitro but nestjs for example

### DISCUSSION THREAD

**Discussion by n4an** [nuxt/nuxt#16722] [Questions]
### Environment

nuxt 3

### Reproduction

---

### Describe the bug

How to use other servers like fastify, nestjs

https://github.com/antoinechalifour/remix-hexagonal-architecture/blob/master/app/routes/register.tsx

**Comment by sevillaarvin** [comment]
Related https://github.com/nuxt/framework/discussions/7872
